qcengine/cli.py

```diff
@@ -4,29 +4,119 @@
 
 import argparse
 import json
+import sys
 
-from . import compute
+from . import compute, compute_procedure  # run and run-procedure
+from . import __version__, list_all_procedures, list_all_programs, list_available_procedures, \
+    list_available_programs, get_procedure, get_program  # info
+from .config import global_repr  # info
 
 __all__ = ["main"]
 
 
 def parse_args():
     parser = argparse.ArgumentParser(description='A CLI for the QCEngine.')
+    subparsers = parser.add_subparsers(dest="command")
 
-    parser.add_argument("program", type=str, default=None, help="The program to call")
-    parser.add_argument("data", type=str, default=None, help="A JSON blob of the task description.")
+    info = subparsers.add_parser('info', help="Print information about QCEngine setup, version, and environment.")
```

Comment:
In QCFractal we use `show`. Any votes for `show` vs `info`? We should be consistent.

Reply:
I think I prefer changing the `qcfractal-server user` CLI to `info` because `qcfractal-server user info` sounds more (English) idiomatic than `qcfractal-server user show`.

Reply:
I was thinking for the `qcfractal-server show ...` commands that either show the current fractal config or alembic info.

This could become `qcfractal-server show programs`. `info` is likely better running through a few examples.

Reply:
`info` was the decision.